modules/miner.go

```diff
@@ -66,8 +70,4 @@ type CPUMiner interface {
 type Miner interface {
 	BlockManager
 	CPUMiner
```

Comment:
Still in transition. Eventually, the server will be reconfigured so that it contains both a block manager and a cpu miner. But that will require making changes to 3-4 files within the api package. Seemed out of scope for this PR, I'm trying to keep things small.

The Miner interface is on the way out, just doing it in steps.